### ISSUE TITLE
Remove 2 more a11y scans 

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#541]
From the HTML/Attribute bundle, remove: 

1) SC 4.1.2 Name, Role, Value	- "This [element type] does not have a name available to an accessibility API. Valid names are [valid names for this element]. [Standard].Principle4.Guideline4_1.4_1_2.H91.[NodeName].Name"	Form control which does not have a valid accessibility API "name". The message states which items count as a valid "name".
2) SC 4.1.2 Name, Role, Value - This [element type] does not have a value available to an accessibility API. Add one [using a element-specific method]. [Standard].Principle4.Guideline4_1.4_1_2.H91.[NodeName].Value"	Form control which does not have a valid accessibility API "value". The message states how to add a valid value.

**Comment by gbinal** [comment]
I've made a pass at this with https://github.com/18F/domain-scan/pull/86 but I'm concerned that it may not quite be formatted correctly.

**Comment by gbinal** [comment]
working on this here - https://github.com/18F/domain-scan/pull/86

**Comment by gbinal** [comment]
Done.
